recipe/meta.yaml

```diff
@@ -23,10 +23,10 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6
-    - google-api-core >=1.21.0,<3.0.0dev
-    - google-auth >=1.24.0,<3.0.0dev
-    - grpcio >=1.8.2,<2.0.0dev
+    - python
+    - google-api-core >=1.31.6,<3.0.0dev
+    - google-auth >=1.25.0,<3.0.0dev
+    - grpcio >=1.38.0,<2.0.0dev
 
 test:
   imports:
```

Comment:
Could you comment:
```
  #downstreams:
    #- gensim
    #- google-cloud-storage
```
because it's good for testing but it's better not to merge it.

Reply:
Sure, I actually wasn't sure what this did so I left it alone.